cranelift/codegen/src/isa/x64/inst.isle

```diff
@@ -2636,6 +2651,26 @@
             (_ Unit (emit (MInst.Mul size signed src1 src2 dst_lo dst_hi))))
         (value_gprs dst_lo dst_hi)))
 
+;; Special case the `mulx` pattern with the BMI2 instruction set.
+(rule 1 (x64_mul (ty_32_or_64 ty) $false src1 src2)
```

Comment:
Is there no `imulx` instruction for signed multiplication?

Reply:
Alas not that I can find, just an unsigned variant. I think it's geared towards bignum libraries where most of the intermediate ops are unsigned ones.

Reply:
Makes sense, thanks for checking